src/components/Header/Header.js

```diff
@@ -12,8 +12,12 @@ export default Header = () => {
     </div>
     <h1 className='header-title'>Dwell-n-Dine</h1>
     <div className='header-buttons'>
-      <button><img src='../../images/favorite-icon.png' alt='favorite icon'></img>Favorites</button>
-      <button><img src='../../images/logout-icon.png' alt='logout icon'></img>Log Out</button>
+        <NavLink to='path-to-logout' className='nav'>
```

Comment:
What's the benefit of using `NavLink` over `Link` from `react-router-dom`? Also, you aren't importing `NavLink` to this file, so make sure you address that.